pubpub/pubpub
# update: non-NodeView compatible Pubs

This PR implements the necessary changes so that `pubpub-editorv6.2.0` can be installed. It adds the necessary functionality to footnote and citations rendering to allow non-nodeview elements. It also makes necessary changes for image/video/iframe breakouts to properly render.

_Test Plan:_
- Pull down `plaindom` branch from `pubpub-editor`.
- Build and link `pubpub-editor` so that the non-nodeview editor code is used. 
- Verify that footnotes, citations, hovering, and server rendering are functional.
- Verify that other elements can be added to the editor.
- Add an image and set it's align option to breakout. Verify that it properly renders full-bleed on desktop and mobile.

_Unhandled by this PR, but necessary soon:_
- Styling of `<PubNoteContent />` component
- Re-design of Citation and Footnote input UI
